Documentation/kernel.md

```diff
@@ -5,7 +5,7 @@ Default Kernel tree: 5.4
 
 | Board | Version |
 |-------|---------|
-| Open Virtual Applicance | 5.4.50 |
+| Open Virtual Appliance | 5.4.50 |
```

Comment:
This file is automatically updated, you need fix in the script too, otherwise the update process fails

Reply:
Resolved, see https://github.com/home-assistant/operating-system/pull/768/commits/0162cbb722d1191dcca644d9a1079900318b0689